tutorials/encode-videos-using-serverless-jobs/index.mdx

```diff
@@ -0,0 +1,134 @@
+---
+meta:
+    title: Encode videos using Serverless Jobs and FFMPEG
+    description: In this tutorial you will learn how to deploy a Serverless Job that encodes videos using FFMPEG
+content:
+    h1: Encode videos using Serverless Jobs and FFMPEG
+    paragraph: In this tutorial you will learn how to deploy a Serverless Job that encodes videos using FFMPEG
+tags: serverless jobs ffmpeg video encoding
+categories:
+  - container-registry
+  - jobs
+  - object-storage
+dates:
+    validation: 2024-05-15
+    posted: 2024-05-15
+---
+
+This tutorial demonstrates the process of encoding videos retrieved from Object Storage using Serverless Jobs: media encoding is a resource-intensive tasks over prolonged durations, making it suitable for Serverless Jobs. The job takes a video file as its input, encodes it using a Docker image based on [FFMPEG](https://ffmpeg.org/), then uploads the encoded video back to the S3 bucket.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [Object Storage bucket](/storage/object/how-to/create-a-bucket/) 
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
```

Comment:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- An [Object Storage bucket](/storage/object/how-to/create-a-bucket/) 
- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
- Installed [MinIO Client](/storage/object/api-cli/installing-minio-client/)
- Installed [Docker engine](https://docs.docker.com/engine/install/)
```

Reply:
What do you mean by installed? Those are used in a Docker image being built part of this tutorial.

Reply:
My bad on the MinIO client, just added Docker as a requirement